### PR TITLE
[@types/node] Update global types

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/api/globals.html#class-abortsignal
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/node/v16/globals.d.ts [hunk 1]
```diff
@@ -56,13 +56,16 @@ interface AbortController {
     abort(reason?: any): void;
 }
 
+import { EventTarget as _EventTarget } from "node:events";
```
Comment (Semigradsky) [inline types/node/v16/globals.d.ts:59]: > Module '"node:events"' has no exported member 'EventTarget'.ts(2305)

Comment (codershiba) [reply types/node/v16/globals.d.ts:59]: Yeah, this is wrong. Should this be done in a way similar to how it's done in node 18 and LTS types?

Comment (Semigradsky) [reply types/node/v16/globals.d.ts:59]: I think yes

**Review by Semigradsky — APPROVED** [review]

**Review by navya9singh — APPROVED** [review]

**Comment by codershiba** [comment]
Ready to merge

**Comment by versat** [comment]
With this change we now get the following errors:
```
node_modules/@types/node/globals.d.ts:60:31 - error TS2304: Cannot find name 'EventTarget'.
60 interface AbortSignal extends EventTarget {
                                 ~~~~~~~~~~~
node_modules/@types/node/globals.d.ts:66:49 - error TS2304: Cannot find name 'Event'.
66     onabort: null | ((this: AbortSignal, event: Event) => any);
```
Is this intended? What is the best way to fix this? Sorry, I am no TS expert.

**Comment by codershiba** [comment]
@Semigradsky created a PR to fix this.

**Comment by peterblazejewicz** [comment]
x-ref: #68250

**Comment by Semigradsky** [comment]
@versat try with new version (`@types/node` >= [16.18.73](https://www.npmjs.com/package/@types/node/v/16.18.73))

**Pull request merged**